infomaniak-build-tools/macos/Readme.md

```diff
@@ -220,9 +223,10 @@ In the project build settings, paste the following lines in the Initial Configur
 -DCMAKE_PREFIX_PATH:STRING=%{Qt:QT_INSTALL_PREFIX}
 -DCMAKE_C_COMPILER:STRING=%{Compiler:Executable:C}
 -DCMAKE_CXX_COMPILER:STRING=%{Compiler:Executable:Cxx}
--DAPPLICATION_CLIENT_EXECUTABLE=kdrive_client
--DKDRIVE_THEME_DIR=/Users/<user>/Projects/kdrive/infomaniak
--DCMAKE_INSTALL_PREFIX=/Users/<user>/Projects/build-kdrive-Qt_6_2_3_for_macOS-Debug/bin
+-DAPPLICATION_CLIENT_EXECUTABLE=kdrive
+-DSOCKETAPI_TEAM_IDENTIFIER_PREFIX=<team id>
+-DKDRIVE_THEME_DIR=/Users/<user>/Projects/desktop-kDrive/infomaniak
```

Comment:
I am wondering if we can use environment variables inside these `CMake` variables. It would be handy to replace `/Users/<user>` with `$HOME` or `%{home}`. 

**Disclaimer.** I didn't manage to make it work reliably.